src/Component/Core/Vendor/Resources/routing/messaging.yaml

```
@@ -0,0 +1,46 @@
+open_marketplace_vendor_messaging_conversation_index:
+    path: /conversations
+    methods: [GET]
+    defaults:
+        _controller:  bitbag.open_marketplace.component.core.vendor.controller.messaging.list_threads
+        _sylius:
+            template: "Context/Vendor/Conversation/index.html.twig"
+
+open_marketplace_vendor_messaging_conversation_show:
+    path: /conversations/{id}
+    methods: [GET,POST]
+    defaults:
+        _controller: bitbag.open_marketplace.component.core.common.controller.messaging.show_thread
+        _sylius:
+            template: "Context/Vendor/Conversation/show.html.twig"
+
+open_marketplace_vendor_messaging_conversation_message_add:
+    path: /conversations/{id}/message/add
+    methods: [GET, POST]
+    defaults:
+        _controller: bitbag.open_marketplace.component.core.common.controller.messaging.create_message
+        _sylius:
+            redirect: open_marketplace_vendor_messaging_conversation_show
+            mail_redirect: open_marketplace_vendor_messaging_conversation_show
+
+open_marketplace_vendor_messaging_conversation_create:
+    path: /conversation/create
+    methods: [GET,POST]
```

GET on a create action sounds stupid
```suggestion
    path: /conversations
    methods: [POST]
```

---

It's also used in GUI as GET endpoint